frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -4,13 +4,15 @@ import * as classNames from 'classnames';
 import { History, Location } from 'history';
 import { useTranslation } from 'react-i18next';
 import { Route, Switch, Link, withRouter, match, matchPath } from 'react-router-dom';
+import { ErrorBoundary } from '@console/shared/src/components/error/error-boundary';
+import { useExtensions, HorizontalNavTab, isHorizontalNavTab } from '@console/plugin-sdk';
 
+import { K8sResourceKind, K8sResourceCommon } from '../../module/k8s';
+import { referenceForModel, referenceFor } from '../../module/k8s/k8s';
+import { ErrorBoundaryFallback } from '../error';
 import { EmptyBox, LoadingBox, StatusBox } from './status-box';
```

Comment:
Arranging as per the lint rules for imports, this import should be at the last, shouldn't it?

Reply:
Good point. Rearrange is now done by eslint.